server/monitor.go

```diff
@@ -146,6 +147,7 @@ func (s *Server) HandleConnz(w http.ResponseWriter, r *http.Request) {
 	// Now we have the real number of ConnInfo objects, we can set c.NumConns
 	// and allocate the array
 	c.NumConns = len(pairs)
+	c.TotalLive = len(s.clients)
```

Comment:
No, this would not be safe since not protected by `s.mu`. Use `totalClients` that was set to `len(s.clients)` above under `s.mu` lock.